octoprint_octorelay/__init__.py

```diff
@@ -310,11 +310,12 @@ def print_started(self):
             settings = self.get_settings_defaults()[index]
             settings.update(self._settings.get([index]))
 
+            relay_pin = int(settings["relay_pin"])
+            inverted = settings['inverted_output']
             autoONforPrint = settings['autoONforPrint']
-            if autoONforPrint:
-                relay_pin = int(settings["relay_pin"])
-                inverted = settings['inverted_output']
-
+            active = settings["active"]
+            if autoONforPrint and active:
+                self._logger.debug(f"turning on pin: {relay_pin}, index: {index}")
```

Comment:
this syntax required Python 3.6 and it has not been used in the currently existing code.
[PEP 498](https://www.python.org/dev/peps/pep-0498/)

Is it OK for the plugin compatibility, @hcooper ?

Reply:
For some reason, the plugin declares the support of Python 2.7.
But probably this is another issue that should be fixed.

<img width="248" alt="Screenshot 2023-07-17 at 23 12 11" src="https://github.com/borisbu/OctoRelay/assets/13189514/f2271e88-45ce-4094-84fe-5c71c28ab186">

Reply:
Well, the compatibility is declared here:

https://github.com/borisbu/OctoRelay/blob/b08a0e4a1d9f38114e312acfc09f704929d4c349/octoprint_octorelay/__init__.py#L417

Reply:
Changing the Python compatibility must be a major version bump according to SemVer.
I'd like to keep this PR on a patch level.
In this regard, I'd like to request rewriting this line in a similar way it's implemented in `print_stopped` method (using `.format()` instead of `f""`),
@hcooper

Reply:
```suggestion
                self._logger.debug("turning on pin: {}, index: {}".format(relay_pin, index))
```